src/HDWallet.cpp

```diff
@@ -32,42 +33,55 @@ HDNode getMasterNode(const HDWallet& wallet, TWCurve curve);
 const char* curveName(TWCurve curve);
 } // namespace
 
-HDWallet::HDWallet(int strength, const std::string& passphrase)
-    : seed(), mnemonic(), passphrase(passphrase) {
+HDWallet::HDWallet(int strength, const std::string& password)
+    : password(password) {
```

Comment:
not sure why change `passphrase` to `password` it's the term used in bip39 with `mnemonic`

Reply:
My rationale was that recovery phrase and passphrase ('phrase' in both) is easy to confuse; moreover phrase is usually a sentence with multiple words and separator spaces, which is true for mnemonic (=recovery phrase) but generally not for the password.  So I think passphrase is more prone to confusion, password is more clear.
E.g. https://en.bitcoinwiki.org/wiki/Mnemonic_phrase uses 'password'.
But you are right, the official BIP39 uses 'passphrase'.
I will keep it as it is.